code/widgets/org.eclipse.scout.jswidgets.ui.html/src/main/js/wrappedform/WrappedFormLookupCall.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2022 BSI Business Systems Integration AG.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Distribution License v1.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/org/documents/edl-v10.html
+ *
+ * Contributors:
+ *     BSI Business Systems Integration AG - initial API and implementation
+ */
+import {ImageField, StaticLookupCall} from '@eclipse-scout/core';
+import {ImageFieldForm, SmartFieldForm, StringFieldForm} from '../index';
+
+export default class WrappedFormLookupCall extends StaticLookupCall {
+
+  constructor() {
+    super();
+  }
+
+  _data() {
+    return WrappedFormLookupCall.DATA;
+  }
+
+  static DATA = [
+    [StringFieldForm, 'StringField'],
```

Comment:
This will bloat up the ui, please use MiniForm and LifecycleForm instead, maybe there is another one.